templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -165,7 +165,7 @@ Keyboard navigation (e.g. arrow keys) works as expected.
 
 * **Toggle shortcuts help**: `?`
 
-* **Toggle gear menu**: `g`
+* **Toggle avatar menu**: `g`
```

Comment:
Can you do `git grep -i `gear menu` change the text to `avatar menu` in other docs too? (like `full-text-search` and `manual-testing`?

Reply:
Sure, will do that.